slack-api-client/src/main/java/com/slack/api/methods/impl/AsyncRateLimitExecutor.java

```diff
@@ -60,7 +62,9 @@ public static AsyncRateLimitExecutor getOrCreate(MethodsClientImpl client, Slack
             Map<String, String> params,
             AsyncExecutionSupplier<T> methodsSupplier) {
         String token = params.get("token");
-        final String teamId = token != null ? teamIdCache.lookupOrResolve(token) : null;
+        final String teamId = (token != null
+                && METHOD_NAMES_TO_SKIP_TEAM_ID_CACHE_RESOLUTION.contains(methodName)) ?
```

Comment:
I see that the other checks against `METHOD_NAMES_TO_SKIP_TEAM_ID_CACHE_RESOLUTION` use the negation `!`, so just wanted to verify that this was intended to be the opposite.

Reply:
Honestly I haven't verified the async client and overlooked this bug. Thanks for the great catch!